packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -178,7 +182,7 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
 export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> implements IFluidContainer {
     private readonly connectedHandler = () => this.emit("connected");
     private readonly disconnectedHandler = () => this.emit("disconnected");
-    private readonly disposedHandler = () => this.emit("disposed");
+    private readonly disposedHandler = (error?: ICriticalContainerError) => this.emit("dispose", error);
```

Comment:
Also - I see the error emitted is changing from "disposed" to "dispose", is that intended?

Reply:
Good catch. I intended to adapt the interface (the two are misaligned), and leave the event the same. So the other way around. But now thinking about it, does it really matter? Either way it is a breaking change. We should still fix it as part of this PR though. The way things stand today ("disposed" is emitted), there is no way to listen to this event without typescript throwing error stating you are listening on the event that given class does not emit (it expects "dispose"). 

Maybe reversing it may appear as a lesses of an issue, and navigate this as a bug fix in short term.

Reply:
Reversed the change.